src/pregame.py

```diff
@@ -303,32 +315,45 @@ def start(var, wrapper, *, forced=False, restart=""):
                 temp_rolesets.append(temp)
         possible_rolesets = temp_rolesets
 
+    if ingame_state.current_mode.CUSTOM_SETTINGS._overridden and not restart:  # Custom settings
+        need_reset = True
+        wvs = sum(addroles[r] for r in Wolfchat)
+        if len(villagers) < (sum(addroles.values()) - sum(addroles[r] for r in ingame_state.current_mode.SECONDARY_ROLES)):
+            wrapper.send(messages["too_few_players_custom"])
+        elif not wvs:
+            wrapper.send(messages["need_one_wolf"])
+        elif wvs > (len(villagers) / 2):
+            wrapper.send(messages["too_many_wolves"])
+        else:
+            need_reset = False
+
+        if need_reset:
+            wrapper.send(messages["default_reset"])
+            stop_game(ingame_state, abort=True, log=False)
+            return
+
```

Comment:
This was actually a conflict of adding it when it didn't exist in master. I don't know what the appropriate solution is, so I'm going to leave the modified PR version in for now.